api/RTCPeerConnection.json

```diff
@@ -176,10 +176,12 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "22",
+              "notes": "Starting in Firefox 68, the <code>candidate</code> parameter is optional when calling the constructor. A <code>null</code> value for <code>candidate</code> indicates no more candidates will be sent, while an empty <code>candidate</code> string indicates that no more candidates will be sent for the current generation of candidates."
             },
             "firefox_android": {
-              "version_added": "44"
+              "version_added": "44",
+              "notes": "Starting in Firefox 68, the <code>candidate</code> parameter is optional when calling the constructor. A <code>null</code> value for <code>candidate</code> indicates no more candidates will be sent, while an empty <code>candidate</code> string indicates that no more candidates will be sent for the current generation of candidates."
```

Comment:
Here too, maybe this note belongs on the constructor instead?

Reply:
Same as above.